EntranceShuffle.py

```diff
@@ -122,18 +126,50 @@ def connect_fairy(world, entrancename, exitname):
                                         ('Stock Pot Inn Secret Entrance', 'Stock Pot Inn'),
                                         ('Mayors Office', 'Mayors Office'),
                                         ('Bomber Bouncer', 'Bomber Tunnel'),
-                                        ('Honey and Darling Exit', 'Clock Town'),
-                                        ('Treasure Chest Shop Exit', 'Clock Town'),
-                                        ('Town Shooting Gallery Exit', 'Clock Town'),
-                                        ('Milk Bar Exit', 'Clock Town'),
-                                        ('Stock Pot Inn Roof', 'Clock Town'),
-                                        ('Stock Pot Inn Front Door', 'Clock Town'),
-                                        ('Mayors Office Exit', 'Clock Town'),
-                                        ('Bomber Tunnel Exit', 'Clock Town'),
+                                        ('Honey and Darling Exit', 'East Clock Town'),
+                                        ('Treasure Chest Shop Exit', 'East Clock Town'),
+                                        ('Town Shooting Gallery Exit', 'East Clock Town'),
+                                        ('Milk Bar Exit', 'East Clock Town'),
+                                        ('Stock Pot Inn Roof', 'East Clock Town'),
+                                        ('Stock Pot Inn Front Door', 'East Clock Town'),
+                                        ('Mayors Office Exit', 'East Clock Town'),
+                                        ('Bomber Tunnel Exit', 'East Clock Town'),
                                         ('ECT Top Exit to SCT', 'South Clock Town'),
                                         ('ECT Bottom Exit to SCT', 'South Clock Town'),
                                         ('ECT Exit to NCT', 'North Clock Town'),
-                                        ('Clock Town East Gate', 'Termina Field')
+                                        ('Clock Town East Gate', 'Termina Field'),
+                                        ('Curiosity Shop', 'Curiosity Shop'),
+                                        ('Trading Post', 'Trading Post'),
+                                        ('Bomb Shop', 'Bomb Shop'),
+                                        ('Post Office', 'Post Office'),
+                                        ('Lottery Shop', 'Lottery Shop'),
```

Comment:
This is slightly confusing. Maybe keep to the convention of naming the first `Entrance` or `Exit` for clarity?